common/src/main/java/com/sequenceiq/cloudbreak/co2/model/ClusterCO2Dto.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.cloudbreak.co2.model;
+
+import java.util.List;
+
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+
+public class ClusterCO2Dto {
+
+    private CloudPlatform cloudPlatform;
+
+    private String region;
+
+    private String status;
+
+    private List<InstanceGroupCO2Dto> instanceGroups;
+
+    public boolean isComputeRunning() {
+        return !status.startsWith("STOPPED") && !status.startsWith("DELETED");
```

Comment:
Is this for ClusterStatus?
Do we query "DELETED" clusters? That should not even be fetch at the moment.